tests/Integrations/Guzzle/V6/GuzzleIntegrationTest.php

```diff
@@ -25,7 +25,7 @@ class GuzzleIntegrationTest extends IntegrationTestCase
     public static function setUpBeforeClass()
     {
         parent::setUpBeforeClass();
-        IntegrationsLoader::load();
+        GuzzleIntegration::load();
```

Comment:
This switch basically means `curl` isn't loaded, I think? Is that the purpose here?

Reply:
Yes that is the purpose. The curl integration is what is making the tests pass currently, by only loading the guzzle integration the test starts failing.